osbenchmark/worker_coordinator/runner.py

```diff
@@ -1212,7 +1238,9 @@ def status(v):
             # we're good with any count of relocating shards.
             expected_relocating_shards = sys.maxsize
 
+        request_context_holder.on_client_request_start()
```

Comment:
@IanHoang @saimedhi We also have custom runners in workloads eg: https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/9c0fdca67b0a929a7f28649c911737898609cb50/nyc_taxis/workload.py#L3

Shouldn't we update there as well?

Reply:
I am not completely sure, but I tested running benchmarks against the 'nyc_taxis' workload and everything seems to be working fine.